### PR TITLE
Update readme description

### PR DESCRIPTION
**Pull request by BobyMCbobs**
Improves README.md to reflect the project.

### CODE REVIEW

**Comment by BobyMCbobs** [comment]
cc @ameukam @BenTheElder

**Comment by BobyMCbobs** [comment]
Is it possible to also update the current project description on the side
> This project exists to explore the creation of a proxy for OCI images. This project is experimental and may change drastically or be cancelled with little or no public notice. 

via a PR?

I'd like to change it to
> This project is the repo for registry.k8s.io, the production domain redirect service for Kubernetes container image artifacts.

File: README.md [hunk 1]
```diff
@@ -16,4 +16,4 @@ You can reach the maintainers of this project at:
 Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
 
 [owners]: https://git.k8s.io/community/contributors/guide/owners.md
-[Creative Commons 4.0]: https://git.k8s.io/website/LICENSE
+[creative commons 4.0]: https://git.k8s.io/website/LICENSE
```
Comment (ameukam) [inline README.md:19]: `Creative Commons` is the right way to write the name. You'll notice https://git.k8s.io/website/LICENSE the `C` are in uppercase.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# oci-proxy
+# registry.k8s.io
 
-This project is currently an experiment, for more see [cmd/archeio](./cmd/archeio/README.md)
+The production redirect domain for Kubernetes artifacts, for more see [cmd/archeio](./cmd/archeio/README.md)
```
Comment (ameukam) [inline README.md:3]:
```suggestion
The production redirector service for container images of the Kubernetes Project, for more see [cmd/archeio](./cmd/archeio/README.md)
```
we only serve container images at the moment.

File: README.md (hunk 1 above)
Comment (BobyMCbobs) [reply README.md]: This is a result of my linter. Reverted in e0679f7

File: README.md (hunk 2 above)
Comment (BobyMCbobs) [reply README.md]: I updated the description in e0679f7. 
I've gone with 
> The production redirector service for Kubernetes container image artifacts

I think it's more concise.

**Comment by ameukam** [comment]
/lgtm
/approve

Thanks!

**Comment by BobyMCbobs** [comment]
> /lgtm /approve
> 
> Thanks!

@ameukam, thank you!

**Pull request merged**